rust/rope/src/lib.rs

```diff
@@ -27,10 +27,14 @@
 extern crate bytecount;
 extern crate memchr;
 extern crate regex;
-extern crate serde;
 extern crate unicode_segmentation;
+
+#[cfg(feature = "dese")]
+extern crate serde;
 #[macro_use]
+#[cfg(feature = "dese")]
```

Comment:
hmmm I think maybe we want this above `#[macro_use]`?

Reply:
Sure